tests/Fixer/Phpdoc/PhpdocVarWithoutNameFixerTest.php

```diff
@@ -503,6 +503,50 @@ trait StaticExample {
     public static $static = "foo";
 }',
             ],
+            'testFixClosureWithThis' => [
```

Comment:
```suggestion
            'complex type with union containing callable that has `$this` in signature' => [
```
Cases' names should be descriptive and friendly for `--testdox` output.

Reply:
The name will be however inconsistent with other namings here. Is that really preferred?

Reply:
When reviewing file I've unfolded file content above your change and found `const are not handled by this fixer` case, that's why I commented, but you're right - most of the cases in this file has camelCase notation. I think we need to convert them anyway, so let's add new cases with proper description.